orga/tests/acceptance/login_test.js

```diff
@@ -0,0 +1,68 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { click, currentURL } from '@ember/test-helpers';
+import { visit } from '@1024pix/ember-testing-library';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import setupIntl from '../helpers/setup-intl';
+
+module('Acceptance | Login', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks, ['fr', 'en']);
+
+  module('when current url does not contain french tld (.fr)', function () {
+    module('when accessing the login page with "Français" as default language', function () {
+      test('displays the login page with "Français" as selected language', async function (assert) {
+        // when
+        const screen = await visit('/connexion');
+
+        // then
+        assert.strictEqual(currentURL(), '/connexion');
+        assert.dom(screen.getByRole('heading', { name: this.intl.t('pages.login.title'), level: 1 })).exists();
+        assert.dom(screen.getByRole('button', { name: 'Français' })).exists();
+      });
+
+      module('when the user select "English" as his language', function () {
+        test('displays the login page with "English" as selected language', async function (assert) {
+          // when
+          const screen = await visit('/connexion');
+          await click(screen.getByRole('button', { name: 'Français' }));
+          await screen.findByRole('listbox');
+          await click(screen.getByRole('option', { name: 'English' }));
+
+          // then
+          assert.strictEqual(currentURL(), '/connexion');
+          assert.dom(screen.getByRole('heading', { name: this.intl.t('pages.login.title'), level: 1 })).exists();
+          assert.dom(screen.getByRole('button', { name: 'English' })).exists();
+        });
```

Comment:
Suggestion : ajouter un test qui vérifie qu'en cliquant, on a bien changé la langue (l'url aura changé ou le texte)

Reply:
Autre remarque, c'est pas déjà testé dans le test d'intégration le "displays the login page with "English" as selected language" ?

Reply:
C'est déjà ce qu'on verifie dans ce test, mais comme on choisit de tester sur des clés de traduction, c'est vrai qu'on ne voit pas le changement de langue. 

Et puis l'url ne change plus quand on passe sur de l'anglais. A la limite on peut passer un query params dans le when et verifier dans le then qu'il n'est plus là 
`const screen = await visit('/connexion?lang=fr')`

Reply:
Le "English" est un justement là pour montrer le changement de langue quand on clique

Reply:
merci pour le /connexion,
c'est pas hyper clair je trouve ce qu'apporte le assert.dom(screen.getByRole('button', { name: 'English' })).exists(); en plus de ce qui est fait en intégration : 
- Qu'on soit en français ou en anglais, c'est English qui s'affiche 
- Dans le premier click de ce test en ligne 29 `await click(screen.getByRole('button', { name: 'Français' }));`, tu testes déjà que française est sélectionné par défaut quand la langue est fr

les tests d'acceptance c'est surtout pour vérifier les changements d'url qu'on pourrait pas testé autrement